src/components/OverviewContent.js

```diff
@@ -1,97 +1,49 @@
-import React, {Component} from "react";
-import {Layout, Divider} from "antd";
+import React, { Component } from "react";
+import { Layout, Divider } from "antd";
 
 import SitePageHeader from "./SitePageHeader";
 import ClinicalSummary from "./overview/ClinicalSummary";
 import VariantsSummary from "./overview/VariantsSummary";
-import OverviewSettingsControl from "./overview/OverviewSettingsControl";
 import { SITE_NAME } from "../constants";
 import ExperimentsSummary from "./overview/ExperimentsSummary";
-import { Modal, Button } from "antd";
 import { DEFAULT_OTHER_THRESHOLD_PERCENTAGE } from "../constants";
 
 class OverviewContent extends Component {
     constructor(props) {
         super(props);
-        const startThreshold =
-          JSON.parse(localStorage.getItem("otherThresholdPercentage")) ?? DEFAULT_OTHER_THRESHOLD_PERCENTAGE;
+        const threshold =
```

Comment:
Consider wrapping this in a try/catch as JSON.parse is susceptible to throw errors when the string is malformed. (in worst case scenario, this could prevent the application to start properly, even when the page is reloaded)

Reply:
addressed above